rpc/src/v1/impls/chain.rs

```diff
@@ -300,8 +300,8 @@ where
         Ok(self.client.block(&BlockId::Hash(block_hash)).map(|block| block.transactions_count()))
     }
 
-    fn get_pending_transactions(&self) -> Result<Vec<Transaction>> {
-        Ok(self.client.ready_transactions().into_iter().map(|signed| signed.into()).collect())
+    fn get_pending_transactions(&self, from: Option<u64>, to: Option<u64>) -> Result<Vec<Transaction>> {
+        Ok(self.client.ready_transactions(from, to).into_iter().map(|signed| signed.into()).collect())
```

Comment:
Please use https://doc.rust-lang.org/std/ops/struct.Range.html
```rust
from.unwrap_or(0)..to.unwrap_or(::std::u64::MAX)
```

Reply:
Thank you, I didn't know about Rust Range